app/views/projects/samples/deletions/destroy.turbo_stream.erb

```diff
@@ -10,9 +10,4 @@
       open: false,
```

Comment:
I think we should only need to append the `flashes` above if the sample was not deleted, since when it is deleted we set the flash and message in the controller

Reply:
Fixed in https://github.com/phac-nml/irida-next/pull/679/commits/bf16319308574e206fac01e46fc034837f02d228